front/src/common/redux/tasks/task.api.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+
+async function getTasks(payload) {
```

Comment:
Pass actual data, not the whole action object

Reply:
done